src/main/java/edu/berkeley/cs186/database/query/QueryPlan.java

```diff
@@ -576,8 +576,22 @@ private QueryOperator addEligibleSelections(QueryOperator source, int except) {
     public QueryOperator minCostSingleAccess(String table) {
         QueryOperator minOp = new SequentialScanOperator(this.transaction, table);
 
-        // TODO(proj3_part2): implement
-        return minOp;
+        // TODO(proj3_part2): implement DONE
+
+        int ioCost = minOp.estimateIOCost();
```

Comment:
```suggestion
        int minIoCost = minOp.estimateIOCost();
```

최소 비용 Op의 비용이니, 더 의미가 담겼으면 좋겠어요.
이러면 갱신하는 코드가 더 이해하기 쉬워질 것 같아요